core/shared/src/main/scala/zio/stm/STM.scala

```diff
@@ -111,6 +105,20 @@ object STM {
   def flatten[E, A](task: STM[E, STM[E, A]]): STM[E, A] =
     ZSTM.flatten(task)
 
+  /**
+   * @see See [[zio.stm.ZSTM.foldCause]]
+   */
+  def foldCause[E, A, B](v: => STM[E, A])(f: Cause[E] => B, g: A => B)(implicit ev: CanFail[E]): USTM[B] =
```

Comment:
I think I would delete these as they already exist on the `ZSTM` trait.

Reply:
These are "my fault" in a way. I proposed adding these operators for symmetry with ZSTM companion. However, if they aren't here, then they should not be on STM either.

Sorry for causing the confusion :pray:.